README.md

```diff
@@ -137,7 +137,7 @@ for more information on how you can help.
 
 ## License
 
-Crossplane is under the Apache 2.0 license.
+Crossplane is under the [Apache 2.0 license](https://github.com/crossplane/crossplane/blob/master/LICENSE).
```

Comment:
Would it be possible for you to follow the same "named links" pattern as most all other links in this file? See the named links section just a few lines below to show examples of how that pattern works :)

Reply:
If the current fork is 1K commits behind upstream then, should I fork the crossplane repo and create a new PR?